docs/_templates/index.html

```diff
@@ -0,0 +1,302 @@
+                        PyStack reports the stack frames of a running Python process or a Python core dump
+                    </p>
+                    <p>
+                        <a href=" {{ pathto('overview') }} "
+                            class="btn btn-primary filled-pystack btn-lg">Documentation</a>
+                        <a href="https://github.com/bloomberg/pystack/discussions"
+                            class="btn btn-secondary btn-lg">Community</a>
+                    </p>
+                    <div class="mouse"> <span></span> </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- Marketing messaging and featurettes
+      ================================================== -->
+        <!-- Wrap the rest of the page in another container to center all the content. -->
+
+        <div class="container marketing">
+
+            <hr class="featurette-divider">
+
+            <!-- Three columns of text below the carousel -->
+            <div class="row">
+                <div class="col-lg-4">
+                    <img class="rounded-circle rounded-header-image"
+                        src="{{ pathto('_static/images/box.png', 1) }}"
+                        alt="Cardboard box" width="140" height="140">
+                    <h2>Processes and core files</h2>
+                    <p>
+                        No matter the situation, PyStack is your reliable
+                        companion for program analysis. If a process is running
+                        more slowly than you expected, PyStack can tell you what
+                        it's doing. If it's hanging, PyStack can tell you where
+                        it got stuck. If it crashed, PyStack can even tell you
+                        what it was doing from the core dump it left behind.
+                    </p>
+                    <p><a class="btn btn-secondary filled-pystack"
+                        href="{{ pathto('process') }}"
+                        role="button">Get started &raquo;</a></p>
+                </div><!-- /.col-lg-4 -->
+                <div class="col-lg-4">
+                    <img class="rounded-circle rounded-header-image"
+                        src="{{ pathto('_static/images/python.png', 1) }}"
+                        alt="Python logo" width="140" height="140">
+                    <h2>Ready for anything</h2>
+                    <p>
+                        PyStack thrives in extreme conditions, always ensuring
+                        accurate insights. It can handle deep call stacks like
+                        a breeze. It can get you an comprehensive report even if
+                        your Python interpreter is heavily optimized and shipped
+                        without debug information. It even copes with core files
+                        caused by memory corruption that may have trashed some
+                        of the interpreter's data structures.
```

Comment:
The last sentence leans towards alliteration which can degrade readability, and the punch line could be moved to the start, ie:
```suggestion
  Even if memory corruption has trashed some of the interpreter's internal data 
  structures, pystack can handle it!
```

Reply:
Hah, it would have been hard to be more alliterative if I had tried. Good catch!